src/schema/v1/sale/index.ts

```diff
@@ -344,6 +344,7 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
         },
       },
       symbol: { type: GraphQLString },
+      timeZone: { type: GraphQLString },
```

Comment:
Does this work? I think it should be:
```suggestion
timeZone: {
  type: GraphQLString,
  resolve: ({ time_zone }) => time_zone,
},
```

Reply:
(Spacing might be off)

Reply:
Thanks, you are correct! Pushing a fix now.